BlueShift-iOS-SDK/InApps/BlueShiftNotificationViewController.m

```diff
@@ -112,6 +112,35 @@ - (void)loadImageFromURL:(UIImageView *)imageView andImageURL:(NSString *)imageU
     imageView.image = newImage;
 }
 
+- (void)setBackgroundImageFromURL:(UIView *)notificationView {
+    if (notificationView && self.notification.templateStyle && self.notification.templateStyle.backgroundImage &&
+    ![self.notification.templateStyle.backgroundImage isEqualToString:@""]) {
+        NSString *backgroundImageURL = self.notification.templateStyle.backgroundImage;
+        NSData * imageData = [[NSData alloc] initWithContentsOfURL: [NSURL URLWithString: backgroundImageURL]];
+        UIImage *image = [[UIImage alloc] initWithData:imageData];
+        [notificationView setBackgroundColor: [UIColor colorWithPatternImage: image]];
+    }
+}
+
+- (void)setBackgroundColor:(UIView *)notificationView {
+    UIColor *backgroundColor = UIColor.whiteColor;
+    if(self.notification.templateStyle && self.notification.templateStyle.backgroundColor && ![self.notification.templateStyle.backgroundColor isEqualToString:@""]){
+        backgroundColor = [self colorWithHexString:self.notification.templateStyle.backgroundColor];
+    }
+    
```

Comment:
Remove line break, as its small block of code and there is no need for a line break. Check the same for line 140.